lib/Runtime/Library/JsBuiltIn/JsBuiltIn.js.bc.32b.h

```diff
@@ -250,10 +250,10 @@
 namespace Js
 {
     const char Library_Bytecode_JsBuiltIn[] = {
-/* 00000000 */ 0x43, 0x68, 0x42, 0x63, 0xC6, 0x1F, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+/* 00000000 */ 0x43, 0x68, 0x42, 0x63, 0xFA, 0x1B, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
```

Comment:
I'm surprised that there is such a large change here; the size of the bytecode has changed significantly, much more than I would expect for a simple changed comment. It also looks like number of built in properties has changed apparently?

Reply:
Possibly this bytecode needed a regen due to other changes to bytecode format? All I know is that this is the bytecode generated by the regen script and the unit test for bytecode regen is happy. @jackhorton do you know of any factors here?

Reply:
if it used to be a single (two-byte) character, then the byte offsets of everything in the JS from that char afterwards would be moved by two, since now it uses two characters/four bytes.

Reply:
Yes, that makes sense. Jimmy said it looked like the size of the bytecode changed significantly. I haven't confirmed that yet.

Reply:
` 5 files changed, 1251 insertions(+), 1495 deletions(-)` looks like the bytecode got smaller by around 250 lines -- more than would be expected for a two-character change in a comment.

Reply:
I understand now, @MSLaguana was saying that going from C6 to FA was a big jump, I think. Perhaps? Regardless, my Intl PR required a bytecode regen and didn't change JsBuiltIns, so I don't think its in a bad state.

Reply:
Right, going from 0x1fc6 to 0x1bfa is quite sizable, and one of the subsequent changes in the header of the bytecode here is the count of built in properties, which seems to be different. I wouldn't have expected very much to change here from just the small change in a comment, but the fact that the test is passing now suggests that it is now a correct reflection of the generated bytecode, so I'm just a bit confused.

Reply:
I re-ran the bytecode regen. It's consistent. (No new changes.) I'll merge the PR.